tests/features/site_tags.feature

```diff
@@ -20,7 +20,7 @@ Feature: Tagging organizational sites
     When I run "terminus site tags list --site=[[test_site_name]] --org=[[organization_name]] --format=json"
     Then I should get:
     """
-    {"tags":["testtag"]}
+    ["testtag"]
```

Comment:
I think this way is better than the old way. In other commands when the user asks for a list, we do not return a JSON object from the API, but an array.